source/nodejs/adaptivecards-site/themes/adaptivecards/source/css/style.css

```diff
@@ -320,7 +320,7 @@ ul.action-list {
 
 .toc-todo li,
 .toc-todo a {
-  padding-left: 30px;
+  padding-left: 15px;
```

Comment:
why is this change required?

Reply:
This is how the website currently looks:
![image](https://user-images.githubusercontent.com/33772802/142527472-6cc0ed64-8c6e-4b8d-9e8e-ed0b6d5c5882.png)

Reply:
This is how the change in the PR makes it look:
![image](https://user-images.githubusercontent.com/33772802/142527938-22e008cb-3608-442c-9352-241dee2c499f.png)

Issue #6510 states the current website is over indented. I reduced this indentation.